spec/lib/services/dialog_import_service_spec.rb

```diff
@@ -28,7 +29,9 @@
 
     before do
       built_dialog_field = DialogField.create(:name => "dialog_field")
+      built_dialog_field2 = DialogField.create(:name => "dialog_field_2")
       allow(dialog_field_importer).to receive(:import_field).and_return(built_dialog_field)
+      allow(dialog_field_importer).to receive(:import_field).and_return(built_dialog_field, built_dialog_field2)
```

Comment:
I think this should just be a change to the previous line instead of an addition.